### ISSUE TITLE
Error when converting to cell_data_set object for Monocle3 using as.cell_data_set() function

### ISSUE THREAD

**Issue by biojiangke** [satijalab/seurat-wrappers#96]
Recently, the conversion code worked before seems to be broken. The line of code was:

`cells.cds <- as.cell_data_set(cells)`

This worked before but with the most current SeuratWrappers and Monocl3, it gives this error:

```
Error in value[[3L]](cond) : 
  invalid subscript 'e' in 'altExp(<SingleCellExperiment>, type="character", ...)':
  'RNA' not in 'altExpNames(<SingleCellExperiment>)'
```

I've checked the seurat object and everything was fine. It has other assays in addition to 'RNA', such as "SCT", "integrated". I tried those and it threw the same error: XXX (assay name) not in SingleCellExperiment object. 

Does anyone have any suggestions?

The package versions are here:

 [1] zoo_1.8-9                   ComplexHeatmap_2.8.0        ggplot2_3.3.3               dplyr_1.0.6                 monocle3_1.0.0             
 [6] SingleCellExperiment_1.14.1 SummarizedExperiment_1.22.0 GenomicRanges_1.44.0        MatrixGenerics_1.4.0        matrixStats_0.58.0         
[11] Biobase_2.52.0              GenomeInfoDb_1.28.0         IRanges_2.26.0              S4Vectors_0.30.0            BiocGenerics_0.38.0        
[16] sctransform_0.3.2           SeuratWrappers_0.3.0        SeuratDisk_0.0.0.9019       RColorBrewer_1.1-2          SeuratObject_4.0.1         
[21] Seurat_4.0.2                Signac_1.2.1

**Comment by biojiangke** [comment]
The same thing and error message seems to happen to as.SingleCellExperiment() function, too.

**Comment by biojiangke** [comment]
Looks like it was already reported here. https://github.com/satijalab/seurat/issues/4531

**Comment by l-magnificence** [comment]
How it can be solved now in seurat 4.0.2 and SingleCellExperiment 1.14.1?

**Comment by andrewwbutler** [comment]
Hi,

Please try updating to Seurat v4.0.3 and see if that resolved the issue.
